src/SFA.DAS.Admin.Aan.Web/Views/NetworkEvent/EventDescription.cshtml

```diff
@@ -0,0 +1,113 @@
+﻿@using SFA.DAS.Admin.Aan.Web.Infrastructure
+@model SFA.DAS.Admin.Aan.Web.Models.NetworkEvent.CreateEventDescriptionViewModel
+@{
+    ViewData["Title"] = "Create Calendar Event";
+}
+
+<partial name="_validationSummary" />
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+       
+        <form asp-route="@RouteNames.CreateEvent.EventDescription" method="POST">
+        <div class="govuk-form-group">
+            <h1 class="govuk-heading-l">
+                <span class="govuk-caption-l">Create event</span>
+            </h1>
+            <input type="hidden" asp-for="@Model.BackLink"/>
+            <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="@Model.EventOutlineMaxCount">
+                <div esfa-validation-marker-for="@Model.EventOutline" class="govuk-form-group"> 
+                    <span asp-validation-for="@Model.EventOutline" class="govuk-error-message"></span>
+                    <div id="EventOutline">
+                        <h1 class="govuk-label-wrapper">
+                            <label class="govuk-label govuk-label--m" for="eventOutline">
+                               Event outline
+                            </label>
+                        </h1>
+                        <div id="eventOutline-hint" class="govuk-hint">
+                            Short description that outlines the event listing
+                        </div>
+                        <textarea class="govuk-textarea govuk-js-character-count" id="eventOutline" name="eventOutline" rows="5" aria-describedby="eventOutline-info eventOutline-hint"
+                                      sfa-validationerror-class="form-control-error" maxlength="500">@Model.EventOutline</textarea>
```

Comment:
In the validator the maxlength verified against 200 character length, here it is set as 500 ?